index.html

```diff
@@ -3,20 +3,11 @@
 <html>
 <head>
 	<title>ES6 Classes</title>
-	<script src="./dist/canvanoid.bundle.js"></script> 
+	
 </head>
 <body>
 	<canvas id="canvas"></canvas>
-	<script> //"babel src -d dist",
-		window.requestAnimationFrame = window.requestAnimationFrame || window.webkitRequestAnimationFrame || 
-									window.msRequestAnimationFrame || window.mozRequestAnimationFrame;
-					
-		var canvas = document.getElementById('canvas')
-		canvas.width = window.innerWidth - 100; 
-		canvas.height = window.innerHeight - 100; 
 
-		var game = new Canvanoid(canvas); 	
-		game.start();
-	</script>
+	<script src="./dist/canvanoid.bundle.js"></script> 
```

Comment:
Mmmm... is something easy to test, just run a couple of wepback generating to hard disk and check that the hash doesn't change because the content is the same, I will give a try and come back to you.